agent/src/http_server/routes/get_users.rs

```diff
@@ -0,0 +1,25 @@
+use axum::{
+    Json
+};
+use serde::Serialize;
+
+
+#[derive(Serialize, Clone)]
```

Comment:
You'll probably want to deserialize this object as some point
```suggestion
#[derive(Serialize, Deserialize, Clone)]
```

Reply:
same